packages/commons-ui-core/src/NavList/NavMenu.js

```diff
@@ -0,0 +1,83 @@
+import { Grid, Typography } from "@mui/material";
+import Button from "@mui/material/Button";
+import { useTheme, styled } from "@mui/material/styles";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavTypography = styled(Typography)({
+  "&:hover": {
+    textDecoration: "underline",
+    color: "blue",
+  },
+});
+
+const NavContainer = styled(Grid)({
+  background: "white",
+});
+
+const NavButton = styled(Button)({
+  boxShadow: "none",
+  textTransform: "capitalize",
+  fontSize: 16,
+  color: "black",
+  margin: 16,
+  backgroundColor: "transparent",
+  borderColor: "none",
+  "&:hover, &:active, &:focus, &:focus-within": {
+    backgroundColor: "transparent",
+    borderColor: "none",
+    boxShadow: "none",
+  },
+});
+
+function NavMenu({ children, menu, buttonVariant, typographyVariant, footer }) {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
+
+  if (!menu?.length) {
+    return null;
+  }
+
+  return (
+    <NavContainer
+      container
+      direction={isMobile || footer ? "column" : "row"}
+      alignItems="flex-start"
+      justifyContent={isMobile ? "flex-start" : "flex-end"}
+    >
+      {menu.map((item) => (
+        <Grid item key={item.label}>
+          <NavButton variant={buttonVariant} href={item.href}>
+            <NavTypography variant={typographyVariant}>
```

Comment:
I don't think we need both. `Link` component is built on [top](https://mui.com/material-ui/react-link/#basic-links) of `Typography`.